db/models.py

```diff
@@ -1 +1,57 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return (f"name: {self.name}\n"
+                f"description: {self.description}")
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(
+        "kind of bonus players can get from it",
+        max_length=255)
```

Comment:
Do not forget to place the closing parenthesis `)` on a new line in case you start a new line after opening the parenthesis `(`. Add this to your code.